src/waltz/quic/fd_quic.c

```diff
@@ -1443,19 +1436,33 @@ fd_quic_handle_v1_initial( fd_quic_t *               quic,
           new_conn_id.conn_id,
           FD_QUIC_MAX_CONN_ID_SZ );
 
+      fd_quic_conn_id_t retry_src_conn_id[1] = {0};
+
       /* Handle retry if configured. */
-      if (quic->config.retry)
-      {
+      if( quic->config.retry ) {
         fd_quic_metrics_t * metrics = &quic->metrics;
 
         /* This is the initial packet before retry. */
-        if (initial->token_len == 0)
-        {
+        if( initial->token_len == 0 ) {
+          /* FIXME we pass the "tp" here to allow the function to add the retry_source_connection_ip
+           * transport parameter to be added.
+           * This is incorrect, since:
+           *   1. "tp" is transient, and will likely be modified before it is encoded into a packet
+           *   2. RETRY is supposed to be stateless
```

Comment:
ah got it thanks

Reply:
That comment needs fixing. I believe the "tp" usage may have been correct. Either way "FIXME" isn't needed anymore